### PR TITLE
Update Changelog/deps and add command to add debug admin

### LINKED ISSUE #1436: Add db util to add an admin to a database file so it can be use for debug an issue

### PR DESCRIPTION
**Pull request by plorenz**
- Add 'add-debug-admin' command. Fixes #1436
- Update changelog

### CODE REVIEW

File: ziti/cmd/database/add_debug_admin.go [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+	stores, err := persistence.NewBoltStores(dbProvider)
+	action.noError(err)
+
+	id := "7dbd3fc9-e4c8-489a-ab8f-4bbb3d768f57"
+	ctx := change.New().SetChangeAuthorType("cli.debug-db").NewMutateContext()
+	err = dbProvider.GetDb().Update(ctx, func(ctx boltz.MutateContext) error {
+		tx := ctx.Tx()
+		identity, _ := stores.Identity.LoadOneById(tx, id)
+		if identity == nil {
```
Comment (andrewpmartinez) [inline ziti/cmd/database/add_debug_admin.go:93]: Does it make sense for this to update the existing one if present?

File: ziti/cmd/database/add_debug_admin.go [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+				IdentityTypeId: "Default",
+				IsDefaultAdmin: false,
+				IsAdmin:        true,
+			}
+			if err = stores.Identity.Create(ctx, identity); err != nil {
+				return err
+			}
+
+			authHandler := model.AuthenticatorManager{}
+			result := authHandler.HashPassword("admin")
```
Comment (andrewpmartinez) [inline ziti/cmd/database/add_debug_admin.go:106]: Why is the password here hard coded?

File: ziti/cmd/database/add_debug_admin.go [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+			authenticator := &persistence.AuthenticatorUpdb{
+				Authenticator: persistence.Authenticator{
+					BaseExtEntity: boltz.BaseExtEntity{
+						Id: eid.New(),
+					},
+					Type:       "updb",
+					IdentityId: id,
+				},
+				Username: "admin",
```
Comment (andrewpmartinez) [inline ziti/cmd/database/add_debug_admin.go:115]: I do not suggest these hard-coded values. I would expect to be able to supply my own username/password. Even in debugging scenarios. 

I understand this is meant for local debugging, but there is a chance people will use this in production environments.

**Review by andrewpmartinez — CHANGES_REQUESTED** [review]
See hardcoded/default values comments.

File: ziti/cmd/database/add_debug_admin.go [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+	"github.com/openziti/ziti/controller/network"
+	"github.com/openziti/ziti/controller/persistence"
+	"github.com/spf13/cobra"
+)
+
+func NewAddDebugAdminAction() *cobra.Command {
+	action := &addDebugAdminAction{}
+	return &cobra.Command{
+		Use:   "add-debug-admin </path/to/ziti-controller.db.file>",
+		Short: "Adds an admin (username: admin, password: admin) to the given database file for debugging purposes",
```
[the remark this answers is not in the document]
Comment (plorenz) [reply ziti/cmd/database/add_debug_admin.go]: fixed

File: ziti/cmd/database/add_debug_admin.go (hunk 1 above)
Comment (plorenz) [reply ziti/cmd/database/add_debug_admin.go]: I changed it to delete the existing identity if a debug-admin already exists.

File: ziti/cmd/database/add_debug_admin.go (hunk 3 above)
Comment (plorenz) [reply ziti/cmd/database/add_debug_admin.go]: Removed hardcoded values

**Review by andrewpmartinez — APPROVED** [review]

**Pull request merged**